airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/dataflow/stats/MetricTracker.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2025 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.load.dataflow.stats
+
+import jakarta.inject.Singleton
+import java.util.concurrent.ConcurrentHashMap
+
+@Singleton
+class MetricTracker {
+
+    private val metrics: MutableMap<String, Double> = ConcurrentHashMap()
```

Comment:
Given that we're tracking a set of known metrics and we should always publish 0, should we just make this a class instead?

Reply:
@gosusnp Maybe...hadn't gotten that far yet.  We should either a) declare all of the metrics in the tracker/some other class that the tracker can access and initialize them to 0 on creation or b) make the caller initialize them.  Seems like A makes more sense and roughly follows what we do in the platform the the metric registry.

Reply:
@gosusnp I took a swing at adding a metric registry (name to be determined) that handles the initialization to 0 for each known metric.